.github/workflows/test_rust.yml

```diff
@@ -153,6 +156,53 @@ jobs:
         env:
           RUSTDOCFLAGS: -D warnings
 
+  coverage:
+    needs: changes
+    if: needs.changes.outputs.should_run == 'true'
+    name: Coverage Report
+    runs-on: ubuntu-24.04
+    steps:
+      - name: Clone Ruffle repo
+        uses: actions/checkout@v4
+
+      - name: Install Rust toolchain
+        uses: dtolnay/rust-toolchain@stable
+        with:
+          # We need nightly for coverage due to:
+          #  1. wayland-backend requiring that,
+          #  2. branch coverage being unstable.
+          #
+          # Ad 1.
+          #   See <https://github.com/Smithay/wayland-rs/commit/05d8307ea647053b949357eb8588dfdab394f4fe>
+          # Ad 2.
+          #   See <https://github.com/taiki-e/cargo-llvm-cov/issues/8>
+          #   See <https://github.com/rust-lang/rust/issues/79649>
+          toolchain: nightly
+          components: llvm-tools
+
+      - name: Install llvm-cov
```

Comment:
nit: Isn't this just the cargo subcommand, with the llvm-cov tool itself already installed above?